### ISSUE TITLE
Trying to get property 'cvterm_id' of non-object for is_a relationship

### ISSUE THREAD

**Issue by abretaud** [tripal/tripal#683]
## BUG/ERROR report

Another warning I get when starting up a tripal 3.x instance:

```
Building cvterm paths for vocabulary: tripal_contact
Clearing the cvtermpath table for this vocabulary...
Clearing completed.
Retrieving relationships...
Processing terms...
Trying to get property 'cvterm_id' of non-object                        [notice]
tripal_chado.cv.api.inc:531
```
[In the code](https://github.com/tripal/tripal/blob/7.x-3.x/tripal_chado/api/modules/tripal_chado.cv.api.inc#L530) the "$is_a" looks strange, but changing it to 'is_a' everywhere doesn't fix the issue

### System information

* Tripal Version: 3.x
* Drupal Version: 7.59
* PostgreSQL Version: 9.5
* PHP Version: 7.x

#### Error messages and screenshots

```
error message
```

**Comment by spficklin** [comment]
@abretaud. That's an unfortunate typo but it does not break the code because it's just storing into the cache a value with the key named '$is_a', although it should be corrected.  

But, thanks for pointing this out. I think the problem is that the fixes to the OBO Importer adds an 'is_a' term for every vocabulary.  But if you are upgrading Tripal from 2.x it may skip already loaded ontologies and the is_a doesn't exist.  We can fix that.  Can you confirm you are upgrading from a 2.x site?

**Comment by abretaud** [comment]
It's not an update, I'm just testing the docker-tripal image with the latest code
(I don't think I'll need to update my instances, I'll just redeploy them and reload the data)

**Comment by spficklin** [comment]
Okay, so it's a clean install.  Are you sure the Tripal Contact vocabulary got loaded?  It seems the Tripal Pub vocabulary may not be loaded as per issue #682

**Comment by abretaud** [comment]
Looking at the log in #682, the contact vocabulary is also loaded

**Comment by spficklin** [comment]
I am not able to duplicate this problem, or the problem described in #682.  I did try to clone and rebuild the docker-tripal project to see if I could duplicate the problem there.  I am able to deal with relatively simple docker setups, that one was a bit more complicated and I couldn't determine where the problem was happening.   

Because this problem does not occur on our unit testing, and a fresh install tested by me cannot duplicate it I removed the release blocker status for this.    

@abretaud did the merged PR #685 fix this problem for you?
